cmd/config.go

```diff
@@ -0,0 +1,89 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/pkg/errors"
+	"github.com/redhat-developer/odo/pkg/config"
+	"github.com/spf13/cobra"
+)
+
+// configurationCmd represents the app command
+var configurationCmd = &cobra.Command{
+	Use:   "config",
+	Short: "Modifies configuration settings",
+	Long: `Modifies Odo specific configuration settings within the config file.
+   # Available Parameters:
```

Comment:
This looks odd when doing --help

```sh
github.com/redhat-developer/odo  pr_589 ✔                                                                                                                                                                                                                                   3d  ⍉
▶ ./odo utils config --help
Modifies Odo specific configuration settings within the config file.
   # Available Parameters:
     UpdateNotification - Controls if an update notification is shown or not (true or false)

Usage:
  odo utils config [flags]
  odo utils config [command]

Aliases:
  config, configuration


Examples:
  # For viewing the current configuration
    odo utils config view
  # Set a configuration value
    odo utils config set UpdateNotification false


Available Commands:
  set         Set a value in odo config file
  view        View current values

Flags:
  -h, --help   help for config

Global Flags:
      --alsologtostderr                  log to standard error as well as files
      --log_backtrace_at traceLocation   when logging hits line file:N, emit a stack trace (default :0)
      --log_dir string                   If non-empty, write log files in this directory
      --logtostderr                      log to standard error instead of files (default false)
      --skip-connection-check            Skip cluster check
      --stderrthreshold severity         logs at or above this threshold go to stderr (default 2)
  -v, --v Level                          log level for V logs
      --vmodule moduleSpec               comma-separated list of pattern=N settings for file-filtered logging

Use "odo utils config [command] --help" for more information about a command.
```

you should instead remove the `#` and remove the tabbing. Also add a newline between the 1st sentence and Available parameters.